src/app/machines/views/MachineList/MachineListControls/MachineActionMenu/MachineActionMenu.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import type { Meta } from "@storybook/react";
+
+import MachineActionMenu from ".";
+
+const meta: Meta<typeof MachineActionMenu> = {
+  title: "Sections/Machine/MachineActionMenu",
+  component: MachineActionMenu,
+  tags: ["autodocs"],
+};
+export default meta;
+
+export const Example = {
+  args: {
+    hasSelection: true,
+  },
```

Comment:
Is it possible to set `setSidePanelContent` here as just a void function? This would avoid error messages that display when clicked in the storybook

Reply:
Thanks for pointing this out!

I'd be cautious when introducing additional manual handlers for props, especially when it's writing additional code that has no other function other than avoiding an error on interaction that we don't really support anyway at the moment.

I'll resolve this in a followup PR by addressing state settters (set* type of props) globally.